lib/src/util.dart

```diff
@@ -68,9 +68,11 @@ class Util {
 
   static int toInt(UtilFunc<ffi.Int32> getFunc) {
     final ptr = ffi.allocate<ffi.Int32>();
-    call(() => getFunc(ptr));
+    int rv;
+    call(() => rv = getFunc(ptr));
```

Comment:
we could make `Util.call()` return the function's return value so that this becomes:

```
if (call(() => getFunc(ptr)) != sp_return.SP_OK) {
  return null;
}
...
```